.github/workflows/release.yml

```diff
@@ -2,7 +2,7 @@ name: Release Changelog
 
 # Only release on a new tag that is a version number.
 on:
-  create:
+  push:
     tags:
```

Comment:
I don't understand why this isn't working as-is with the `create` event. I would have thought the `tags` condition would prevent this job from running until a tag is manually created - and a version tag is only created on merge into main, right?

Reply:
https://github.com/actions/runner/issues/1007

Reply:
ah that explains it lol thanks